### PR TITLE
Added new Resource Files

### PR DESCRIPTION
**Pull request by rachelbock**
- Cleaned up data and resource files to only contain what the application needs.
- Added climbs and projects data classes.
- Created a ProjectResource class to get project data from the Clamber database.
- Updated the WallsResource class to get the Wall and WallSection data from the database and return the appropriate
  wall sections and climbs from the database to the android application.

### CODE REVIEW

File: src/main/java/com/rachelbock/data/Project.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.rachelbock.data;
+
+/**
+ * Created by rage on 4/4/16.
```
Comment (nonvirtualthunk) [inline src/main/java/com/rachelbock/data/Project.java]: javadoc

File: src/main/java/com/rachelbock/resources/ProjectResource.java [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+/**
+ * Class to set up the database connection for projects
+ */
+
+@Path("/projects/{user_name}")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class ProjectResource {
+
+    public Connection getConnection() throws SQLException {
```
Comment (nonvirtualthunk) [inline src/main/java/com/rachelbock/resources/ProjectResource.java:28]: Probably getting to the point where we should talk about using a connection pool, that can wait for the next PR though

File: src/main/java/com/rachelbock/resources/ProjectResource.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+        String serverName = "localhost";
+        int portNumber = 3306;
+        String dbName = "clamber";
+
+        String connectionString = "jdbc:" + dbms + "://" + serverName + ":" + portNumber + "/" + dbName;
+        Connection conn = DriverManager.getConnection(connectionString, connectionProps);
+        return conn;
+    }
+
+    @GET
```
Comment (nonvirtualthunk) [inline src/main/java/com/rachelbock/resources/ProjectResource.java:52]: Probably a good idea to start adding javadocs to the GET methods, though these are fairly straightforward. Something along the lines of "Retrieves climb data for all climbs that the given user has marked as being a project. Returns an empty list if that user has no projects, or if the user does not exist." should be sufficient. The idea being to make it clear what data it's returning, and how it handles edge cases.

File: src/main/java/com/rachelbock/resources/ProjectResource.java [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        } catch (SQLException e1) {
+            e1.printStackTrace();
+        }
+
+        throw new InternalServerErrorException("Could not create new project for user " + request.getUsername());
+    }
+
+
+/**
+ * Defines how we expect the json in the body to look
```
Comment (nonvirtualthunk) [inline src/main/java/com/rachelbock/resources/ProjectResource.java]: indentation

File: src/main/java/com/rachelbock/resources/WallsResource.java [hunk 5]
```diff
@@ -1,28 +1,25 @@
...
 
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Properties;
 
 /**
  * Created by rage on 3/13/16.
```
Comment (nonvirtualthunk) [inline src/main/java/com/rachelbock/resources/WallsResource.java]: Javadoc!

File: src/main/java/com/rachelbock/resources/WallsResource.java [hunk 6]
```diff
@@ -1,28 +1,25 @@
...
  */
-@Path("/walls/{id}")
+@Path("/user/{username}/walls/{id}")
```
Comment (nonvirtualthunk) [inline src/main/java/com/rachelbock/resources/WallsResource.java]: should probably change `{id}` to something more specific

**Comment by nonvirtualthunk** [comment]
I think all the logic is solid, just add those docs and I'm satisfied

File: src/main/java/com/rachelbock/data/Project.java (hunk 1 above)
Comment (rachelbock) [inline src/main/java/com/rachelbock/data/Project.java]: done

File: src/main/java/com/rachelbock/resources/ProjectResource.java (hunk 3 above)
Comment (rachelbock) [inline src/main/java/com/rachelbock/resources/ProjectResource.java:52]: Done

File: src/main/java/com/rachelbock/resources/ProjectResource.java (hunk 4 above)
Comment (rachelbock) [inline src/main/java/com/rachelbock/resources/ProjectResource.java]: fixed

File: src/main/java/com/rachelbock/resources/WallsResource.java (hunk 6 above)
Comment (rachelbock) [inline src/main/java/com/rachelbock/resources/WallsResource.java]: updated to wall_id

**Pull request merged**